input/elasticapm/result.go

```diff
@@ -23,20 +23,34 @@ import (
 
 type Result struct {
 	errorsSpace [5]error
+
 	// Errors holds a limited number of errors that occurred while
 	// processing the event stream. If the limit is reached, the
 	// counters above are still incremented.
 	Errors []error
+
 	// Accepted holds the number of valid events accepted.
-	Accepted int
+	Accepted        int
+	AcceptedDetails ProcessedDetail
```

Comment:
We still should add a godoc comment for new `AcceptedDetails` field here

Reply:
Done! Thanks